src/packages/ratmama/ratmama_parser.py

```diff
@@ -122,9 +124,12 @@
 
         ctx.bot.board.append(rescue, overwrite=False)
         index = ctx.bot.board.find_by_name(client=client_name).board_index
+        landmark = await Galaxy.find_nearest_landmark((await Galaxy.find_system_by_name(system_name)))
```

Comment:
[Error] This method call is risky, because `find_system_by_name` can return `None`, which will cause `find_nearest_landmark` to break.